components/time-picker/time-picker.component.spec.ts

```diff
@@ -10,8 +10,6 @@ import { NoopAnimationsModule } from '@angular/platform-browser/animations';
 import { dispatchFakeEvent, dispatchMouseEvent, typeInElement } from 'ng-zorro-antd/core/testing';
 import { PREFIX_CLASS } from 'ng-zorro-antd/date-picker';
 import { getPickerInput } from 'ng-zorro-antd/date-picker/testing/util';
-import { registerLocaleData } from '@angular/common';
```

Comment:
Sorry, our import order rule looks like it's not working, this problem has been fixed on master, please rebase onto master again. :trollface:

Reply:
haha, no problem